bom/models.py

```diff
@@ -216,21 +216,20 @@ def increment_str(s):
     def __str__(self):
         return u'%s' % (self.full_part_number())
 
-class ChangeHistory(models.Model):
+
+class PartChangeHistory(models.Model):
     part_id = models.CharField(max_length=2)
```

Comment:
Any reason `part_id` is a CharField? I'd recommend changing this line from:
`part_id = models.CharField(max_length=2)`
to
`models.ForeignKey(Part, on_delete=models.CASCADE)`

Reply:
Hi Mike, Just submitted the below diff to convert part_id to Foreign Key. Let me know if you have more feedback.